system-addon/content-src/styles/_theme.scss

```diff
@@ -77,7 +78,8 @@ body {
   --newtab-link-secondary-color: $pocket-teal;
   --newtab-text-conditional-color: $grey-10;
   --newtab-text-primary-color: $grey-10;
-  --newtab-text-secondary-color: $grey-10-40;
+  --newtab-text-secondary-color: $grey-10-80;
+  --newtab-text-tertiary-color: $grey-10-40;
```

Comment:
I would think whatever contrast test is failing for existing usage of secondary-color would then continue to fail for the context now using tertiary color. If it's a usability issue, why is it okay to let the context disappear?